### ISSUE TITLE
Color Contrast ratio for copy right text in About Xamarin Workbooks window is not having 4.5:1 ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#380]
### Steps to Reproduce

1. Launch Xamarin Workbooks application.
2. Select platform to WPF and create workbook.
3. Navigate to Help->About Xamarin workbooks and invoke it.
4. Verify the color contrast ratio of copy right text.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1 ratio
### Actual Behavior
Color contrast ratio for copy right text is 3.9:1
### User Impact
Low vision user will feel difficult to see the text.
### Screenshot
![about xamarin workbooks](https://user-images.githubusercontent.com/22956130/40762821-dc54f8cc-64bf-11e8-98a2-233810fae1bf.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
[‎30-‎05-‎2018 18:20]  Vamsi Krishna Velichety (Wipro Ltd.):  
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS1.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
